### PR TITLE
made required changes

### PR DESCRIPTION
**Pull request by Sarahkhan20**

### CODE REVIEW

File: SE_AIDS_S2.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+      "lectureStartTime": "12:45 pm",
+      "lectureEndTime": "1:45 pm"
+    },
+    {
+      "lectureName": "EM ",
+      "lectureFacultyName": "Poonam Soni",
+      "lectureBatch": "All",
+      "lectureStartTime": "1:45 pm",
+      "lectureEndTime": "2:45 pm"
+    }
```
Comment (prasaddd77) [inline SE_AIDS_S2.json:190]: After EM IV lecture there are labs for all 3 batches.

File: SE_IT_S1.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+      {
+         "lectureName": "PY labs",
+         "lectureFacultyName": "Sachi Natu",
+         "lectureBatch": "S11",
+         "lectureStartTime": "10:30 am",
+         "lectureEndTime": "12:30 pm"
+      },
+      {
+         "lectureName": "PY labs",
+         "lectureFacultyName": "SAG",
```
Comment (prasaddd77) [inline SE_IT_S1.json:186]: SAG is Sheetal Gondal. Write full name.

**Review by prasaddd77 — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by Sarahkhan20** [comment]
Sure

**Comment by prasaddd77** [comment]
SE Timetables Done.

**Pull request merged**